{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Over-Representation Analysis\n",
    "\n",
    "Notebook for performing an over-representation analysis on the top and bottom strains, in terms of redness and normalized redness.\n",
    "\n",
    "2020-02-26\n",
    "\n",
    "## Initial boilerplate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Standard variables loaded, you are good to go!\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "from dotenv import load_dotenv, find_dotenv\n",
    "from os.path import join, dirname, basename, exists, isdir\n",
    "\n",
    "### Load environmental variables from the project root directory ###\n",
    "# find .env automagically by walking up directories until it's found\n",
    "dotenv_path = find_dotenv()\n",
    "\n",
    "# load up the entries as environment variables\n",
    "load_dotenv(dotenv_path)\n",
    "\n",
    "# now you can get the variables using their names\n",
    "\n",
    "# Check whether a network drive has been specified\n",
    "DATABASE = os.environ.get(\"NETWORK_URL\")\n",
    "if DATABASE == 'None':\n",
    "    pass\n",
    "else:\n",
    "    pass\n",
    "    #mount network drive here\n",
    "\n",
    "# set up directory paths\n",
    "CURRENT_DIR = os.getcwd()\n",
    "PROJ = dirname(dotenv_path) # project root directory\n",
    "\n",
    "DATA = join(PROJ, 'data') #data directory\n",
    "RAW_EXTERNAL = join(DATA, 'raw_external') # external data raw directory\n",
    "RAW_INTERNAL = join(DATA, 'raw_internal') # internal data raw directory\n",
    "INTERMEDIATE = join(DATA, 'intermediate') # intermediate data directory\n",
    "FINAL = join(DATA, 'final') # final data directory\n",
    "\n",
    "RESULTS = join(PROJ, 'results') # output directory\n",
    "FIGURES = join(RESULTS, 'figures') # figure output directory\n",
    "PICTURES = join(RESULTS, 'pictures') # picture output directory\n",
    "\n",
    "\n",
    "# make folders specific for certain data\n",
    "folder_name = ''\n",
    "if folder_name != '':\n",
    "    #make folders if they don't exist\n",
    "    if not exists(join(RAW_EXTERNAL, folder_name)):\n",
    "        os.makedirs(join(RAW_EXTERNAL, folder_name))\n",
    "\n",
    "    if not exists(join(INTERMEDIATE, folder_name)):\n",
    "        os.makedirs(join(INTERMEDIATE, folder_name))\n",
    "\n",
    "    if not exists(join(FINAL, folder_name)):\n",
    "        os.makedirs(join(FINAL, folder_name))\n",
    "\n",
    "print('Standard variables loaded, you are good to go!')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Load data\n",
    "\n",
    "We will use:\n",
    "* 4 lists (generated manually):\n",
    "  * Top 500 genes in terms of redness, with size $\\geq$ 180 and count > 1.\n",
    "  * Top 500 genes in terms of normalized redness, with size $\\geq$ 260.\n",
    "  * Bottom 500 genes in terms of redness, with size $\\geq$ 180 and count > 1.\n",
    "  * Bottom 500 genes in terms of normalized redness, with size $\\geq$ 180 and count > 1.\n",
    "* The lookup table connecting positions in the 384-well plate to gene ids, as it has all GO-terms associated to each gene ID."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    top_redness top_norm_redness bottom_redness bottom_norm_redness\n",
      "0          CSE2             GAS5          IBA57                OPY1\n",
      "1          MLC2             YOR1           OPY1                RIM8\n",
      "2          YOR1          YJL175W        YCR090C               IBA57\n",
      "3          NTA1             SKY1           HTD2                BUD8\n",
      "4       YOL153C          YIL096C           DIA3               RPL9A\n",
      "..          ...              ...            ...                 ...\n",
      "495        PTK1             TRP1          SPO74                RTC2\n",
      "496     YBR287W          YPR071W        YDL211C                IPK1\n",
      "497        PLB2             BPT1           YCH1             YMR160W\n",
      "498        TRP1             NEJ1           YSP2                STP2\n",
      "499     YKR070W             REX3        YIL012W             YGL152C\n",
      "\n",
      "[500 rows x 4 columns]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "data = pd.read_csv(join(INTERMEDIATE,\"top_and_bottom_500.csv\"))\n",
    "print(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "            ORF       Gene                                         Decription  \\\n",
      "0       YLL040C      VPS13  Protein of unknown function; heterooligomeric ...   \n",
      "1       YAL068C       PAU8  Protein of unknown function, member of the ser...   \n",
      "2       YAL067C       SEO1  Putative permease, member of the allantoate tr...   \n",
      "3       YLL041C       SDH2  Iron-sulfur protein subunit of succinate dehyd...   \n",
      "4       YAL066W    YAL066W  Dubious open reading frame unlikely to encode ...   \n",
      "...         ...        ...                                                ...   \n",
      "4842    YER122C       GLO3  ADP-ribosylation factor GTPase activating prot...   \n",
      "4843    YJL075C      APQ13  Dubious open reading frame, unlikely to encode...   \n",
      "4844    YBR035C       PDX3  Pyridoxine (pyridoxamine) phosphate oxidase, h...   \n",
      "4845    YPR072W       NOT5  Subunit of the CCR4-NOT complex, which is a gl...   \n",
      "4846  YOR008C-A  YOR008C-A  Putative protein of unknown function, includes...   \n",
      "\n",
      "                                  GO Biological Process  \\\n",
      "0     transport, sporulation resulting in formation ...   \n",
      "1     sporulation resulting in formation of a cellul...   \n",
      "2                                             transport   \n",
      "3     cofactor metabolic process, generation of prec...   \n",
      "4                                    biological_process   \n",
      "...                                                 ...   \n",
      "4842  transport, vesicle organization, vesicle-media...   \n",
      "4843                                 biological_process   \n",
      "4844                   cellular lipid metabolic process   \n",
      "4845  transcription, protein modification process, R...   \n",
      "4846                                 biological_process   \n",
      "\n",
      "          GO Molecular Function  \\\n",
      "0            molecular_function   \n",
      "1            molecular_function   \n",
      "2          transporter activity   \n",
      "3       oxidoreductase activity   \n",
      "4            molecular_function   \n",
      "...                         ...   \n",
      "4842  enzyme regulator activity   \n",
      "4843         molecular_function   \n",
      "4844    oxidoreductase activity   \n",
      "4845            ligase activity   \n",
      "4846         molecular_function   \n",
      "\n",
      "                                  GO Cellular Component  \n",
      "0                    mitochondrion, membrane, cytoplasm  \n",
      "1                                    cellular_component  \n",
      "2                                              membrane  \n",
      "3     mitochondrion, membrane, mitochondrial envelop...  \n",
      "4                                    cellular_component  \n",
      "...                                                 ...  \n",
      "4842  endomembrane system, cytoplasmic membrane-boun...  \n",
      "4843                                 cellular_component  \n",
      "4844                                 cellular_component  \n",
      "4845                                 nucleus, cytoplasm  \n",
      "4846                                 cellular_component  \n",
      "\n",
      "[4847 rows x 6 columns]\n"
     ]
    }
   ],
   "source": [
    "gene_ids = pd.read_csv(join(RAW_EXTERNAL,\"geneIDs.txt\"), sep=\"\\t\")\n",
    "gene_ids = gene_ids[gene_ids[\"Gene\"] != \"Blank\"]  # filter out blank data\n",
    "gene_ids = gene_ids.drop(columns=[\"Plate #\", \"Row\", \"Column\", \"96-position\"])  # remove unused columns\n",
    "gene_ids.index = range(0,len(gene_ids.index))  # re-index for consistency later on\n",
    "print(gene_ids)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. GO terms\n",
    "\n",
    "We will first collect all GO terms in a single list, to be able to iterate through later."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['-', 'DNA metabolic process', 'RNA metabolic process', 'biological_process', 'cell budding', 'cell cycle', 'cellular amino acid and derivative metabolic process', 'cellular aromatic compound metabolic process', 'cellular carbohydrate metabolic process', 'cellular component morphogenesis', 'cellular homeostasis', 'cellular lipid metabolic process', 'cellular membrane organization', 'cellular protein catabolic process', 'cellular respiration', 'chromosome organization', 'chromosome segregation', 'cofactor metabolic process', 'conjugation', 'cytokinesis', 'cytoskeleton organization', 'fungal-type cell wall organization', 'generation of precursor metabolites and energy', 'heterocycle metabolic process', 'meiosis', 'mitochondrion organization', 'nucleus organization', 'other', 'peroxisome organization', 'protein complex biogenesis', 'protein folding', 'protein modification process', 'pseudohyphal growth', 'response to chemical stimulus', 'response to stress', 'ribosome biogenesis', 'signal transduction', 'sporulation resulting in formation of a cellular spore', 'transcription', 'translation', 'transport', 'transposition', 'vacuole organization', 'vesicle organization', 'vesicle-mediated transport', 'vitamin metabolic process']\n"
     ]
    }
   ],
   "source": [
    "GOterms = []\n",
    "for (idx, row) in gene_ids.iterrows():\n",
    "    row[\"GO Biological Process\"] = row[\"GO Biological Process\"].split(\", \")\n",
    "    GOterms.extend(row[\"GO Biological Process\"])\n",
    "GOterms = list(set(GOterms))\n",
    "GOterms.sort()\n",
    "print(GOterms)\n",
    "GOterms.remove('-')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Over-Representation Analysis\n",
    "\n",
    "We need a function for the over-representation analysis. For each GO term the Fisher matrix is constructed as:\n",
    "\n",
    "|   | Selected in group | Not selected |\n",
    "|:-:|:-----------------:|:------------:|\n",
    "| With GO term | F[0][0] | F[0][1] |\n",
    "| Without GO term | F[1][0] | F[1][1] |\n",
    "\n",
    "And then the alternative hypothesis is that there is a higher percentage of genes in the selected group with the GO term Vs without the GO term."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from fisher import pvalue\n",
    "\n",
    "# over-representation function:\n",
    "def over_rep(data, group_name, gene_ids, GOterms):\n",
    "    pvalues = pd.DataFrame()\n",
    "    group = list(data[group_name])\n",
    "    for GOterm in GOterms:\n",
    "        F = [[0, 0], [0, 0]]\n",
    "        for (idx, row) in gene_ids.iterrows():\n",
    "            i = 0 if GOterm in row[\"GO Biological Process\"] else 1\n",
    "            j = 0 if row[\"Gene\"] in group else 1\n",
    "            F[i][j] += 1\n",
    "        pval = pvalue(F[0][0], F[0][1], F[1][0], F[1][1]).right_tail\n",
    "        new_row = pd.DataFrame(data={\"genes.selected\":F[0][0], \"genes.total\":F[0][0] + F[0][1], \"p.value\": pval}, index=[GOterm])\n",
    "        pvalues = pvalues.append(new_row)\n",
    "    print(pvalues[pvalues[\"p.value\"] < 0.05])\n",
    "    return pvalues"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we can call the function for all 4 groups:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [genes.selected, genes.total, p.value]\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "p_values_top_redness = over_rep(data, \"top_redness\", gene_ids, GOterms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                    genes.selected  genes.total   p.value\n",
      "biological_process             173         1405  0.004948\n"
     ]
    }
   ],
   "source": [
    "p_values_top_norm_redness = over_rep(data, \"top_norm_redness\", gene_ids, GOterms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                              genes.selected  genes.total   p.value\n",
      "protein modification process              56          410  0.026396\n"
     ]
    }
   ],
   "source": [
    "p_values_bottom_redness = over_rep(data, \"bottom_redness\", gene_ids, GOterms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                           genes.selected  genes.total   p.value\n",
      "vitamin metabolic process               9           46  0.049671\n"
     ]
    }
   ],
   "source": [
    "p_values_bottom_norm_redness = over_rep(data, \"bottom_norm_redness\", gene_ids, GOterms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Results don't look very promising... most likely because there are not many GO terms included in the original file.\n",
    "\n",
    "## 4. Using GOATOOLS\n",
    "\n",
    "Let's try using more GO terms, via GOATOOLS. For that, we first need Entrez IDs for all genes both from the total list of _S. cerevisiae_ genes and from the 4 selected groups. We can get them using the `mygene` package ([Wu et al. 2014](https://www.biorxiv.org/content/10.1101/009332v1)), querying the original ORFs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "querying 1-1000...done.\n",
      "querying 1001-2000...done.\n",
      "querying 2001-3000...done.\n",
      "querying 3001-4000...done.\n",
      "querying 4001-4847...done.\n",
      "Finished.\n",
      "45 input query terms found dup hits:\n",
      "\t[('YML086C', 2), ('YMR169C', 2), ('YMR174C', 2), ('YMR175W', 2), ('YML035C', 2), ('YMR191W', 2), ('Y\n",
      "411 input query terms found no hit:\n",
      "\t['YAL066W', 'YLL047W', 'YLR123C', 'YLR124W', 'YML089C', 'YAL045C', 'YAL043C-a', 'YML084W', 'YMR151W'\n",
      "Pass \"returnall=True\" to return complete lists of duplicate or missing query terms.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[{'query': 'YLL040C',\n",
       "  '_id': '850619',\n",
       "  '_score': 12.898697,\n",
       "  'entrezgene': '850619'},\n",
       " {'query': 'YAL068C',\n",
       "  '_id': '851229',\n",
       "  '_score': 12.88437,\n",
       "  'entrezgene': '851229'},\n",
       " {'query': 'YAL067C',\n",
       "  '_id': '851230',\n",
       "  '_score': 13.23507,\n",
       "  'entrezgene': '851230'},\n",
       " {'query': 'YLL041C',\n",
       "  '_id': '850685',\n",
       "  '_score': 13.233125,\n",
       "  'entrezgene': '850685'},\n",
       " {'query': 'YAL066W', 'notfound': True},\n",
       " {'query': 'YLL042C',\n",
       "  '_id': '850684',\n",
       "  '_score': 13.22153,\n",
       "  'entrezgene': '850684'},\n",
       " {'query': 'YAL065C',\n",
       "  '_id': '851232',\n",
       "  '_score': 13.740374,\n",
       "  'entrezgene': '851232'},\n",
       " {'query': 'YLL043W',\n",
       "  '_id': '850683',\n",
       "  '_score': 13.2291975,\n",
       "  'entrezgene': '850683'},\n",
       " {'query': 'YAL062W',\n",
       "  '_id': '851237',\n",
       "  '_score': 13.228214,\n",
       "  'entrezgene': '851237'},\n",
       " {'query': 'YLL045C',\n",
       "  '_id': '850682',\n",
       "  '_score': 13.233941,\n",
       "  'entrezgene': '850682'}]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import mygene\n",
    "mg = mygene.MyGeneInfo()\n",
    "geneORFs = list(gene_ids[\"ORF\"])\n",
    "entrezIDs = mg.querymany(geneORFs, scopes='locus_tag', fields='entrezgene', species=559292)  # S. cerevisiae S288C\n",
    "entrezIDs[:10]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note that several entries are either duplicated or don't match to anything. Also, `entrezIDs` is in dictionary format, but we need it as a list of just `entrezgene` ids. So we will create a list with just that (that we will call `population`) and translate all genes in the selected groups:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4436\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>top_redness</th>\n",
       "      <th>top_norm_redness</th>\n",
       "      <th>bottom_redness</th>\n",
       "      <th>bottom_norm_redness</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>855744</td>\n",
       "      <td>854127</td>\n",
       "      <td>853586</td>\n",
       "      <td>852426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>856318</td>\n",
       "      <td>853198</td>\n",
       "      <td>852426</td>\n",
       "      <td>852837</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>853198</td>\n",
       "      <td>YJL175W</td>\n",
       "      <td>850452</td>\n",
       "      <td>853586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>853525</td>\n",
       "      <td>855256</td>\n",
       "      <td>856464</td>\n",
       "      <td>851067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>854012</td>\n",
       "      <td>854711</td>\n",
       "      <td>851537</td>\n",
       "      <td>852730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>853635</td>\n",
       "      <td>851570</td>\n",
       "      <td>852705</td>\n",
       "      <td>852444</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>852590</td>\n",
       "      <td>856184</td>\n",
       "      <td>851315</td>\n",
       "      <td>851910</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>855018</td>\n",
       "      <td>850645</td>\n",
       "      <td>853117</td>\n",
       "      <td>855195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>851570</td>\n",
       "      <td>850970</td>\n",
       "      <td>851926</td>\n",
       "      <td>856397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>853944</td>\n",
       "      <td>850797</td>\n",
       "      <td>854803</td>\n",
       "      <td>YGL152C</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>500 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    top_redness top_norm_redness bottom_redness bottom_norm_redness\n",
       "0        855744           854127         853586              852426\n",
       "1        856318           853198         852426              852837\n",
       "2        853198          YJL175W         850452              853586\n",
       "3        853525           855256         856464              851067\n",
       "4        854012           854711         851537              852730\n",
       "..          ...              ...            ...                 ...\n",
       "495      853635           851570         852705              852444\n",
       "496      852590           856184         851315              851910\n",
       "497      855018           850645         853117              855195\n",
       "498      851570           850970         851926              856397\n",
       "499      853944           850797         854803             YGL152C\n",
       "\n",
       "[500 rows x 4 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "population = []\n",
    "for (idx, row) in gene_ids.iterrows():\n",
    "    gene = entrezIDs[idx]\n",
    "    if \"entrezgene\" in gene:\n",
    "        entrezID = int(gene[\"entrezgene\"])\n",
    "        population.append(entrezID)  # Add to population\n",
    "        data = data.replace(to_replace=row[\"Gene\"], value=entrezID)  # Replace in all selected groups\n",
    "        \n",
    "print(len(population))\n",
    "data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note that not all genes are added to the selected genes, due to missing IDs, but the coverage is good enough (above 90% in all cases):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "top_redness: 462\n",
      "top_norm_redness: 458\n",
      "bottom_redness: 461\n",
      "bottom_norm_redness: 458\n"
     ]
    }
   ],
   "source": [
    "for col in data:\n",
    "    mapped_genes = sum(map(lambda x : type(x) is int, list(data[col])))\n",
    "    print(col + \": \" + str(mapped_genes))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " So let's proceed. We now need a list of all ontologies, and a list of all associations gene/ontology. The package GOATOOLS ([Klopfenstein et al. 2019](https://doi.org/10.1038/s41598-018-28948-z)) provides that, together with improved enrichment tools (corrected p-values, additional tests, etc), so we will from here forward use that."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "requests.get(http://purl.obolibrary.org/obo/go/go-basic.obo, stream=True)\n",
      "  WROTE: go-basic.obo\n",
      "\n",
      "go-basic.obo: fmt(1.2) rel(2020-02-21) 47,245 GO Terms\n",
      "FTP RETR ftp.ncbi.nlm.nih.gov gene/DATA gene2go.gz -> gene2go.gz\n",
      "  gunzip gene2go.gz\n",
      "HMS:0:00:02.819932  94,086 annotations,  6,349 genes,  6,156 GOs, 1 taxids READ: gene2go \n"
     ]
    }
   ],
   "source": [
    "# Get ontologies (from http://geneontology.org/ontology/go-basic.obo)\n",
    "from goatools.base import download_go_basic_obo\n",
    "from goatools.obo_parser import GODag\n",
    "obo_fname = download_go_basic_obo()  # Download data\n",
    "obodag = GODag(obo_fname)  # Load data\n",
    "\n",
    "# Get associations genes <-> ontologies (from ftp://ftp.ncbi.nlm.nih.gov/gene/DATA/gene2go.gz)\n",
    "from goatools.base import download_ncbi_associations\n",
    "from goatools.anno.genetogo_reader import Gene2GoReader\n",
    "gene2go_fname = download_ncbi_associations()  # Download data\n",
    "objanno = Gene2GoReader(gene2go_fname, taxids=[559292])  # Read data (only for Saccharomyces cerevisiae S288C)\n",
    "ns2assoc = objanno.get_ns2assc()\n",
    "\n",
    "#Remove downloaded data:\n",
    "os.remove(obo_fname)\n",
    "os.remove(gene2go_fname)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we create an over-representation object, with all genes, ontologies, associations and settings included in it. This object will be used for every single analysis to perform later on:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Load BP Gene Ontology Analysis ...\n",
      " 88%  3,887 of  4,436 population items found in association\n",
      "\n",
      "Load CC Gene Ontology Analysis ...\n",
      " 93%  4,141 of  4,436 population items found in association\n",
      "\n",
      "Load MF Gene Ontology Analysis ...\n",
      " 75%  3,338 of  4,436 population items found in association\n"
     ]
    }
   ],
   "source": [
    "# Over-representation object:\n",
    "from goatools.goea.go_enrichment_ns import GOEnrichmentStudyNS\n",
    "goeaobj = GOEnrichmentStudyNS(\n",
    "    population,  # Population (all mapped yeast genes in Entrez format)\n",
    "    ns2assoc,  # geneid/GO associations\n",
    "    obodag,  # Ontologies\n",
    "    propagate_counts = False,  # ???\n",
    "    alpha = 0.05,  # default significance cut-off\n",
    "    methods = ['fdr_bh'])  # default multipletest correction method"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Using this object, we can repeat the analysis from before for all 4 groups:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    405 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     902 GO terms are associated with    405 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 91%    418 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     238 GO terms are associated with    418 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 73%    338 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     516 GO terms are associated with    338 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     58 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "def goea(data, group_name):\n",
    "    goea_results_all = goeaobj.run_study(list(data[group_name]))\n",
    "    goea_results_sig = [r for r in goea_results_all if r.p_uncorrected < 0.05]\n",
    "    goea_results_sig.sort(key=lambda x: (x.NS, x.GO))\n",
    "    goeaobj.wr_tsv(join(FINAL, \"p_values_\" + group_name + \".tsv\"), goea_results_sig)\n",
    "\n",
    "goea(data, \"top_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 84%    387 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     876 GO terms are associated with    387 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 92%    423 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     224 GO terms are associated with    423 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 73%    333 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     466 GO terms are associated with    333 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     88 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_norm_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data, \"top_norm_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    406 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     872 GO terms are associated with    406 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 94%    435 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     281 GO terms are associated with    435 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 77%    356 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     479 GO terms are associated with    356 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     84 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data, \"bottom_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    404 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     861 GO terms are associated with    404 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 93%    426 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     282 GO terms are associated with    426 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 76%    349 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     489 GO terms are associated with    349 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     54 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_norm_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data, \"bottom_norm_redness\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As observed, no ontology shows to be significant when looking at the corrected p-values. Some ontologies do show significance if we consider only the uncorrected p-values, but then false positives are to be expected. We should probably play with the number of genes inside of each group, as maybe 500 was too many...\n",
    "\n",
    "## 5. Changing the group size\n",
    "\n",
    "Let's start by lowering the group to 100 genes:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 90%     85 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     289 GO terms are associated with     85 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 91%     86 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      90 GO terms are associated with     86 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 80%     75 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     182 GO terms are associated with     75 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "    122 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data.head(100), \"top_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 92%     83 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     298 GO terms are associated with     83 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 92%     83 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      93 GO terms are associated with     83 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 80%     72 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     144 GO terms are associated with     72 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "    135 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_norm_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data.head(100), \"top_norm_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 85%     77 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     258 GO terms are associated with     77 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 96%     87 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     107 GO terms are associated with     87 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 71%     65 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     125 GO terms are associated with     65 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "    104 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data.head(100), \"bottom_redness\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 84%     77 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     236 GO terms are associated with     77 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 93%     86 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     111 GO terms are associated with     86 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 68%     63 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     128 GO terms are associated with     63 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     90 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_norm_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data.head(100), \"bottom_norm_redness\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Still nothing significant, although the number of uncorrected p-values below 0.05 increases for all conditions. We will then make a function that changes the total size of genes assesed to find something significant:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%     10 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      38 GO terms are associated with     10 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%     10 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      16 GO terms are associated with     10 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%     10 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      37 GO terms are associated with     10 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 10 IDs ...\n",
      " 89%      8 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      43 GO terms are associated with      8 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%      9 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      21 GO terms are associated with      9 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 10 IDs ...\n",
      " 67%      6 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      17 GO terms are associated with      6 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 10 IDs ...\n",
      " 89%      8 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      29 GO terms are associated with      8 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%      9 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      23 GO terms are associated with      9 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 10 IDs ...\n",
      " 89%      8 of      9 study items found in association\n",
      " 90%      9 of     10 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      17 GO terms are associated with      8 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%     10 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      38 GO terms are associated with     10 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 10 IDs ...\n",
      "100%     10 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      34 GO terms are associated with     10 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 10 IDs ...\n",
      " 70%      7 of     10 study items found in association\n",
      "100%     10 of     10 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      15 GO terms are associated with      7 of     10 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 95%     19 of     20 study items found in association\n",
      "100%     20 of     20 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      75 GO terms are associated with     19 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 90%     18 of     20 study items found in association\n",
      "100%     20 of     20 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      33 GO terms are associated with     18 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 90%     18 of     20 study items found in association\n",
      "100%     20 of     20 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      54 GO terms are associated with     18 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 95%     18 of     19 study items found in association\n",
      " 95%     19 of     20 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      82 GO terms are associated with     18 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 95%     18 of     19 study items found in association\n",
      " 95%     19 of     20 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      35 GO terms are associated with     18 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 74%     14 of     19 study items found in association\n",
      " 95%     19 of     20 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      39 GO terms are associated with     14 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 89%     16 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      73 GO terms are associated with     16 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 20 IDs ...\n",
      "100%     18 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      46 GO terms are associated with     18 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 72%     13 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      34 GO terms are associated with     13 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 83%     15 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "      70 GO terms are associated with     15 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 89%     16 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      41 GO terms are associated with     16 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 20 IDs ...\n",
      " 61%     11 of     18 study items found in association\n",
      " 90%     18 of     20 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      38 GO terms are associated with     11 of     20 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 93%     27 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     113 GO terms are associated with     27 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 90%     26 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      42 GO terms are associated with     26 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 86%     25 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      74 GO terms are associated with     25 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 97%     28 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     104 GO terms are associated with     28 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 97%     28 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      43 GO terms are associated with     28 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 79%     23 of     29 study items found in association\n",
      " 97%     29 of     30 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      54 GO terms are associated with     23 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 93%     26 of     28 study items found in association\n",
      " 93%     28 of     30 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     112 GO terms are associated with     26 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 30 IDs ...\n",
      "100%     28 of     28 study items found in association\n",
      " 93%     28 of     30 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      65 GO terms are associated with     28 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 75%     21 of     28 study items found in association\n",
      " 93%     28 of     30 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      47 GO terms are associated with     21 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 88%     23 of     26 study items found in association\n",
      " 87%     26 of     30 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     101 GO terms are associated with     23 of     30 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 88%     23 of     26 study items found in association\n",
      " 87%     26 of     30 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      57 GO terms are associated with     23 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 30 IDs ...\n",
      " 73%     19 of     26 study items found in association\n",
      " 87%     26 of     30 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      61 GO terms are associated with     19 of     30 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 92%     36 of     39 study items found in association\n",
      " 98%     39 of     40 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     154 GO terms are associated with     36 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 92%     36 of     39 study items found in association\n",
      " 98%     39 of     40 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      56 GO terms are associated with     36 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 79%     31 of     39 study items found in association\n",
      " 98%     39 of     40 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      95 GO terms are associated with     31 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 97%     36 of     37 study items found in association\n",
      " 92%     37 of     40 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     136 GO terms are associated with     36 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 97%     36 of     37 study items found in association\n",
      " 92%     37 of     40 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      52 GO terms are associated with     36 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 84%     31 of     37 study items found in association\n",
      " 92%     37 of     40 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      73 GO terms are associated with     31 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 92%     35 of     38 study items found in association\n",
      " 95%     38 of     40 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     139 GO terms are associated with     35 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 97%     37 of     38 study items found in association\n",
      " 95%     38 of     40 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      73 GO terms are associated with     37 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 74%     28 of     38 study items found in association\n",
      " 95%     38 of     40 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      52 GO terms are associated with     28 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 89%     31 of     35 study items found in association\n",
      " 88%     35 of     40 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     132 GO terms are associated with     31 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 91%     32 of     35 study items found in association\n",
      " 88%     35 of     40 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      69 GO terms are associated with     32 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 40 IDs ...\n",
      " 74%     26 of     35 study items found in association\n",
      " 88%     35 of     40 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      72 GO terms are associated with     26 of     40 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 90%     43 of     48 study items found in association\n",
      " 96%     48 of     50 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     171 GO terms are associated with     43 of     50 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 88%     42 of     48 study items found in association\n",
      " 96%     48 of     50 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      58 GO terms are associated with     42 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 77%     37 of     48 study items found in association\n",
      " 96%     48 of     50 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     109 GO terms are associated with     37 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 96%     44 of     46 study items found in association\n",
      " 92%     46 of     50 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     158 GO terms are associated with     44 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 93%     43 of     46 study items found in association\n",
      " 92%     46 of     50 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      55 GO terms are associated with     43 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 83%     38 of     46 study items found in association\n",
      " 92%     46 of     50 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      85 GO terms are associated with     38 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 89%     42 of     47 study items found in association\n",
      " 94%     47 of     50 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     152 GO terms are associated with     42 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 98%     46 of     47 study items found in association\n",
      " 94%     47 of     50 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      78 GO terms are associated with     46 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 74%     35 of     47 study items found in association\n",
      " 94%     47 of     50 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      57 GO terms are associated with     35 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 89%     39 of     44 study items found in association\n",
      " 88%     44 of     50 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     159 GO terms are associated with     39 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 91%     40 of     44 study items found in association\n",
      " 88%     44 of     50 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      77 GO terms are associated with     40 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 50 IDs ...\n",
      " 70%     31 of     44 study items found in association\n",
      " 88%     44 of     50 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      82 GO terms are associated with     31 of     50 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 91%     52 of     57 study items found in association\n",
      " 95%     57 of     60 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     199 GO terms are associated with     52 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 89%     51 of     57 study items found in association\n",
      " 95%     57 of     60 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      69 GO terms are associated with     51 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 79%     45 of     57 study items found in association\n",
      " 95%     57 of     60 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     132 GO terms are associated with     45 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 91%     49 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     176 GO terms are associated with     49 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 91%     49 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      61 GO terms are associated with     49 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 78%     42 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      95 GO terms are associated with     42 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 89%     50 of     56 study items found in association\n",
      " 93%     56 of     60 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     175 GO terms are associated with     50 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 96%     54 of     56 study items found in association\n",
      " 93%     56 of     60 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      81 GO terms are associated with     54 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 71%     40 of     56 study items found in association\n",
      " 93%     56 of     60 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      74 GO terms are associated with     40 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 85%     46 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     168 GO terms are associated with     46 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 93%     50 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      82 GO terms are associated with     50 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 60 IDs ...\n",
      " 67%     36 of     54 study items found in association\n",
      " 90%     54 of     60 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      89 GO terms are associated with     36 of     60 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 90%     60 of     67 study items found in association\n",
      " 96%     67 of     70 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     221 GO terms are associated with     60 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 90%     60 of     67 study items found in association\n",
      " 96%     67 of     70 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      71 GO terms are associated with     60 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 79%     53 of     67 study items found in association\n",
      " 96%     67 of     70 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     148 GO terms are associated with     53 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 92%     59 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     212 GO terms are associated with     59 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 91%     58 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      74 GO terms are associated with     58 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 81%     52 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     113 GO terms are associated with     52 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 86%     55 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     188 GO terms are associated with     55 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 95%     61 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      87 GO terms are associated with     61 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 70%     45 of     64 study items found in association\n",
      " 91%     64 of     70 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      82 GO terms are associated with     45 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 81%     51 of     63 study items found in association\n",
      " 90%     63 of     70 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     177 GO terms are associated with     51 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 94%     59 of     63 study items found in association\n",
      " 90%     63 of     70 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      85 GO terms are associated with     59 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 70 IDs ...\n",
      " 63%     40 of     63 study items found in association\n",
      " 90%     63 of     70 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "      95 GO terms are associated with     40 of     70 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 91%     68 of     75 study items found in association\n",
      " 94%     75 of     80 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     235 GO terms are associated with     68 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 91%     68 of     75 study items found in association\n",
      " 94%     75 of     80 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      79 GO terms are associated with     68 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 79%     59 of     75 study items found in association\n",
      " 94%     75 of     80 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     152 GO terms are associated with     59 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 93%     68 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     245 GO terms are associated with     68 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 92%     67 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      77 GO terms are associated with     67 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 81%     59 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     129 GO terms are associated with     59 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 88%     64 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     209 GO terms are associated with     64 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 96%     70 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      94 GO terms are associated with     70 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 74%     54 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     100 GO terms are associated with     54 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 82%     60 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     199 GO terms are associated with     60 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 95%     69 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      96 GO terms are associated with     69 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 80 IDs ...\n",
      " 67%     49 of     73 study items found in association\n",
      " 91%     73 of     80 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     108 GO terms are associated with     49 of     80 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 89%     75 of     84 study items found in association\n",
      " 93%     84 of     90 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     265 GO terms are associated with     75 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 90%     76 of     84 study items found in association\n",
      " 93%     84 of     90 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      87 GO terms are associated with     76 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 79%     66 of     84 study items found in association\n",
      " 93%     84 of     90 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     168 GO terms are associated with     66 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 93%     76 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     278 GO terms are associated with     76 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 93%     76 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      85 GO terms are associated with     76 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 80%     66 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     139 GO terms are associated with     66 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 87%     71 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     236 GO terms are associated with     71 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 95%     78 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     103 GO terms are associated with     78 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 72%     59 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     114 GO terms are associated with     59 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 83%     68 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     216 GO terms are associated with     68 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 94%     77 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     106 GO terms are associated with     77 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 90 IDs ...\n",
      " 70%     57 of     82 study items found in association\n",
      " 91%     82 of     90 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     116 GO terms are associated with     57 of     90 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 90%     85 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     289 GO terms are associated with     85 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 91%     86 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      90 GO terms are associated with     86 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 80%     75 of     94 study items found in association\n",
      " 94%     94 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     182 GO terms are associated with     75 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 92%     83 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     298 GO terms are associated with     83 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 92%     83 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      93 GO terms are associated with     83 of    100 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 80%     72 of     90 study items found in association\n",
      " 90%     90 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     144 GO terms are associated with     72 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 85%     77 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     258 GO terms are associated with     77 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 96%     87 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     107 GO terms are associated with     87 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 71%     65 of     91 study items found in association\n",
      " 91%     91 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     125 GO terms are associated with     65 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 84%     77 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     236 GO terms are associated with     77 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 93%     86 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     111 GO terms are associated with     86 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 100 IDs ...\n",
      " 68%     63 of     92 study items found in association\n",
      " 92%     92 of    100 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     128 GO terms are associated with     63 of    100 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 90%     94 of    104 study items found in association\n",
      " 95%    104 of    110 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     322 GO terms are associated with     94 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 91%     95 of    104 study items found in association\n",
      " 95%    104 of    110 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     100 GO terms are associated with     95 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 80%     83 of    104 study items found in association\n",
      " 95%    104 of    110 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     197 GO terms are associated with     83 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 92%     91 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     322 GO terms are associated with     91 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 93%     92 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "      93 GO terms are associated with     92 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 80%     79 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     159 GO terms are associated with     79 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 86%     85 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     273 GO terms are associated with     85 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 96%     95 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     113 GO terms are associated with     95 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 73%     72 of     99 study items found in association\n",
      " 90%     99 of    110 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     134 GO terms are associated with     72 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 84%     86 of    102 study items found in association\n",
      " 93%    102 of    110 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     255 GO terms are associated with     86 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 94%     96 of    102 study items found in association\n",
      " 93%    102 of    110 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     112 GO terms are associated with     96 of    110 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 110 IDs ...\n",
      " 71%     72 of    102 study items found in association\n",
      " 93%    102 of    110 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     141 GO terms are associated with     72 of    110 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 91%    104 of    114 study items found in association\n",
      " 95%    114 of    120 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     347 GO terms are associated with    104 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 92%    105 of    114 study items found in association\n",
      " 95%    114 of    120 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     104 GO terms are associated with    105 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 81%     92 of    114 study items found in association\n",
      " 95%    114 of    120 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     206 GO terms are associated with     92 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 92%     99 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     358 GO terms are associated with     99 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 93%    100 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     107 GO terms are associated with    100 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 80%     86 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     171 GO terms are associated with     86 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 87%     94 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     295 GO terms are associated with     94 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 96%    104 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     126 GO terms are associated with    104 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 75%     81 of    108 study items found in association\n",
      " 90%    108 of    120 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     153 GO terms are associated with     81 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 86%     95 of    111 study items found in association\n",
      " 92%    111 of    120 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     281 GO terms are associated with     95 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 95%    105 of    111 study items found in association\n",
      " 92%    111 of    120 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     119 GO terms are associated with    105 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 120 IDs ...\n",
      " 72%     80 of    111 study items found in association\n",
      " 92%    111 of    120 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     160 GO terms are associated with     80 of    120 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 91%    113 of    124 study items found in association\n",
      " 95%    124 of    130 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     370 GO terms are associated with    113 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 93%    115 of    124 study items found in association\n",
      " 95%    124 of    130 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     106 GO terms are associated with    115 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 81%    101 of    124 study items found in association\n",
      " 95%    124 of    130 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     222 GO terms are associated with    101 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 92%    108 of    118 study items found in association\n",
      " 91%    118 of    130 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     387 GO terms are associated with    108 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 93%    110 of    118 study items found in association\n",
      " 91%    118 of    130 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     112 GO terms are associated with    110 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 80%     94 of    118 study items found in association\n",
      " 91%    118 of    130 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     192 GO terms are associated with     94 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 88%    103 of    117 study items found in association\n",
      " 90%    117 of    130 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     320 GO terms are associated with    103 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 97%    113 of    117 study items found in association\n",
      " 90%    117 of    130 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     132 GO terms are associated with    113 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 77%     90 of    117 study items found in association\n",
      " 90%    117 of    130 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     170 GO terms are associated with     90 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 86%    103 of    120 study items found in association\n",
      " 92%    120 of    130 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     308 GO terms are associated with    103 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 94%    113 of    120 study items found in association\n",
      " 92%    120 of    130 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     130 GO terms are associated with    113 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 130 IDs ...\n",
      " 71%     85 of    120 study items found in association\n",
      " 92%    120 of    130 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     168 GO terms are associated with     85 of    130 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 90%    121 of    134 study items found in association\n",
      " 96%    134 of    140 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     399 GO terms are associated with    121 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 93%    124 of    134 study items found in association\n",
      " 96%    134 of    140 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     110 GO terms are associated with    124 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 80%    107 of    134 study items found in association\n",
      " 96%    134 of    140 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     238 GO terms are associated with    107 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 90%    114 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     397 GO terms are associated with    114 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 92%    116 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     116 GO terms are associated with    116 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 79%     99 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     197 GO terms are associated with     99 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 88%    111 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     338 GO terms are associated with    111 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 97%    122 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     143 GO terms are associated with    122 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 76%     96 of    126 study items found in association\n",
      " 90%    126 of    140 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     174 GO terms are associated with     96 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 85%    109 of    128 study items found in association\n",
      " 91%    128 of    140 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     319 GO terms are associated with    109 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 94%    120 of    128 study items found in association\n",
      " 91%    128 of    140 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     133 GO terms are associated with    120 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 140 IDs ...\n",
      " 70%     90 of    128 study items found in association\n",
      " 91%    128 of    140 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     182 GO terms are associated with     90 of    140 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 91%    130 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     414 GO terms are associated with    130 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 93%    133 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     113 GO terms are associated with    133 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 79%    113 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     246 GO terms are associated with    113 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       5 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 90%    122 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     405 GO terms are associated with    122 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 93%    125 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     122 GO terms are associated with    125 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 79%    106 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     202 GO terms are associated with    106 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 88%    120 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     366 GO terms are associated with    120 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 96%    130 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     152 GO terms are associated with    130 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 77%    105 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     190 GO terms are associated with    105 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 85%    117 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     332 GO terms are associated with    117 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 94%    129 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     139 GO terms are associated with    129 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 70%     96 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     193 GO terms are associated with     96 of    150 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 90%    137 of    152 study items found in association\n",
      " 95%    152 of    160 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     433 GO terms are associated with    137 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 93%    142 of    152 study items found in association\n",
      " 95%    152 of    160 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     119 GO terms are associated with    142 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 79%    120 of    152 study items found in association\n",
      " 95%    152 of    160 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     254 GO terms are associated with    120 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 90%    128 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     418 GO terms are associated with    128 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 91%    130 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     122 GO terms are associated with    130 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 78%    111 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     210 GO terms are associated with    111 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 88%    126 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     378 GO terms are associated with    126 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 95%    136 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     157 GO terms are associated with    136 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 78%    111 of    143 study items found in association\n",
      " 89%    143 of    160 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     197 GO terms are associated with    111 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 86%    127 of    147 study items found in association\n",
      " 92%    147 of    160 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     356 GO terms are associated with    127 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 95%    139 of    147 study items found in association\n",
      " 92%    147 of    160 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     145 GO terms are associated with    139 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 160 IDs ...\n",
      " 70%    103 of    147 study items found in association\n",
      " 92%    147 of    160 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     204 GO terms are associated with    103 of    160 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 90%    146 of    162 study items found in association\n",
      " 95%    162 of    170 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     449 GO terms are associated with    146 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 93%    151 of    162 study items found in association\n",
      " 95%    162 of    170 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     130 GO terms are associated with    151 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 77%    125 of    162 study items found in association\n",
      " 95%    162 of    170 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     266 GO terms are associated with    125 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 90%    138 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     453 GO terms are associated with    138 of    170 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 91%    139 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     126 GO terms are associated with    139 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 78%    120 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     223 GO terms are associated with    120 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 89%    136 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     403 GO terms are associated with    136 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 95%    146 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     160 GO terms are associated with    146 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 79%    121 of    153 study items found in association\n",
      " 90%    153 of    170 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     219 GO terms are associated with    121 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 87%    135 of    156 study items found in association\n",
      " 92%    156 of    170 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     371 GO terms are associated with    135 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 95%    148 of    156 study items found in association\n",
      " 92%    156 of    170 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     146 GO terms are associated with    148 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 170 IDs ...\n",
      " 69%    108 of    156 study items found in association\n",
      " 92%    156 of    170 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     211 GO terms are associated with    108 of    170 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 89%    153 of    171 study items found in association\n",
      " 95%    171 of    180 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     463 GO terms are associated with    153 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 93%    159 of    171 study items found in association\n",
      " 95%    171 of    180 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     133 GO terms are associated with    159 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 77%    131 of    171 study items found in association\n",
      " 95%    171 of    180 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     279 GO terms are associated with    131 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 88%    142 of    162 study items found in association\n",
      " 90%    162 of    180 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     459 GO terms are associated with    142 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 91%    148 of    162 study items found in association\n",
      " 90%    162 of    180 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     127 GO terms are associated with    148 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 76%    123 of    162 study items found in association\n",
      " 90%    162 of    180 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     231 GO terms are associated with    123 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 89%    145 of    163 study items found in association\n",
      " 91%    163 of    180 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     445 GO terms are associated with    145 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 96%    156 of    163 study items found in association\n",
      " 91%    163 of    180 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     173 GO terms are associated with    156 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 79%    129 of    163 study items found in association\n",
      " 91%    163 of    180 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     233 GO terms are associated with    129 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 87%    144 of    166 study items found in association\n",
      " 92%    166 of    180 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     388 GO terms are associated with    144 of    180 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 95%    157 of    166 study items found in association\n",
      " 92%    166 of    180 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     152 GO terms are associated with    157 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 180 IDs ...\n",
      " 70%    116 of    166 study items found in association\n",
      " 92%    166 of    180 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     222 GO terms are associated with    116 of    180 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 89%    161 of    180 study items found in association\n",
      " 95%    180 of    190 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     479 GO terms are associated with    161 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 92%    166 of    180 study items found in association\n",
      " 95%    180 of    190 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     134 GO terms are associated with    166 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 77%    138 of    180 study items found in association\n",
      " 95%    180 of    190 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     290 GO terms are associated with    138 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 87%    150 of    172 study items found in association\n",
      " 91%    172 of    190 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     478 GO terms are associated with    150 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 91%    157 of    172 study items found in association\n",
      " 91%    172 of    190 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     140 GO terms are associated with    157 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 76%    130 of    172 study items found in association\n",
      " 91%    172 of    190 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     233 GO terms are associated with    130 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 89%    150 of    169 study items found in association\n",
      " 89%    169 of    190 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     456 GO terms are associated with    150 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 95%    161 of    169 study items found in association\n",
      " 89%    169 of    190 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     175 GO terms are associated with    161 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 79%    133 of    169 study items found in association\n",
      " 89%    169 of    190 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     238 GO terms are associated with    133 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 86%    150 of    174 study items found in association\n",
      " 92%    174 of    190 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     398 GO terms are associated with    150 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 95%    165 of    174 study items found in association\n",
      " 92%    174 of    190 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     156 GO terms are associated with    165 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 190 IDs ...\n",
      " 71%    123 of    174 study items found in association\n",
      " 92%    174 of    190 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     235 GO terms are associated with    123 of    190 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 90%    169 of    188 study items found in association\n",
      " 94%    188 of    200 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     496 GO terms are associated with    169 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 93%    174 of    188 study items found in association\n",
      " 94%    188 of    200 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     139 GO terms are associated with    174 of    200 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 77%    144 of    188 study items found in association\n",
      " 94%    188 of    200 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     296 GO terms are associated with    144 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 87%    158 of    181 study items found in association\n",
      " 90%    181 of    200 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     491 GO terms are associated with    158 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 91%    165 of    181 study items found in association\n",
      " 90%    181 of    200 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     142 GO terms are associated with    165 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 75%    136 of    181 study items found in association\n",
      " 90%    181 of    200 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     238 GO terms are associated with    136 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 88%    157 of    178 study items found in association\n",
      " 89%    178 of    200 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     465 GO terms are associated with    157 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 94%    168 of    178 study items found in association\n",
      " 89%    178 of    200 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     176 GO terms are associated with    168 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 78%    138 of    178 study items found in association\n",
      " 89%    178 of    200 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     252 GO terms are associated with    138 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 86%    158 of    183 study items found in association\n",
      " 92%    183 of    200 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     423 GO terms are associated with    158 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 94%    172 of    183 study items found in association\n",
      " 92%    183 of    200 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     161 GO terms are associated with    172 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 200 IDs ...\n",
      " 71%    130 of    183 study items found in association\n",
      " 92%    183 of    200 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     241 GO terms are associated with    130 of    200 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 90%    178 of    197 study items found in association\n",
      " 94%    197 of    210 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     508 GO terms are associated with    178 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 93%    183 of    197 study items found in association\n",
      " 94%    197 of    210 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     141 GO terms are associated with    183 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 76%    150 of    197 study items found in association\n",
      " 94%    197 of    210 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     302 GO terms are associated with    150 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 88%    167 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     512 GO terms are associated with    167 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 92%    174 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     143 GO terms are associated with    174 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 76%    144 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     249 GO terms are associated with    144 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 88%    165 of    187 study items found in association\n",
      " 89%    187 of    210 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     475 GO terms are associated with    165 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 95%    177 of    187 study items found in association\n",
      " 89%    187 of    210 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     180 GO terms are associated with    177 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 78%    146 of    187 study items found in association\n",
      " 89%    187 of    210 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     259 GO terms are associated with    146 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 87%    165 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     435 GO terms are associated with    165 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 94%    179 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     166 GO terms are associated with    179 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 210 IDs ...\n",
      " 72%    136 of    190 study items found in association\n",
      " 90%    190 of    210 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     251 GO terms are associated with    136 of    210 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 91%    187 of    206 study items found in association\n",
      " 94%    206 of    220 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     526 GO terms are associated with    187 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 93%    191 of    206 study items found in association\n",
      " 94%    206 of    220 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     144 GO terms are associated with    191 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 76%    157 of    206 study items found in association\n",
      " 94%    206 of    220 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     315 GO terms are associated with    157 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 88%    174 of    198 study items found in association\n",
      " 90%    198 of    220 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     520 GO terms are associated with    174 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 92%    182 of    198 study items found in association\n",
      " 90%    198 of    220 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     143 GO terms are associated with    182 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 76%    150 of    198 study items found in association\n",
      " 90%    198 of    220 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     255 GO terms are associated with    150 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 89%    174 of    196 study items found in association\n",
      " 89%    196 of    220 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     490 GO terms are associated with    174 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 95%    186 of    196 study items found in association\n",
      " 89%    196 of    220 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     182 GO terms are associated with    186 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 78%    153 of    196 study items found in association\n",
      " 89%    196 of    220 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     269 GO terms are associated with    153 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 87%    173 of    199 study items found in association\n",
      " 90%    199 of    220 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     451 GO terms are associated with    173 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 94%    187 of    199 study items found in association\n",
      " 90%    199 of    220 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     167 GO terms are associated with    187 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 220 IDs ...\n",
      " 72%    144 of    199 study items found in association\n",
      " 90%    199 of    220 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     268 GO terms are associated with    144 of    220 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 90%    193 of    214 study items found in association\n",
      " 93%    214 of    230 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     539 GO terms are associated with    193 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 92%    196 of    214 study items found in association\n",
      " 93%    214 of    230 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     147 GO terms are associated with    196 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 76%    162 of    214 study items found in association\n",
      " 93%    214 of    230 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     322 GO terms are associated with    162 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 87%    181 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     533 GO terms are associated with    181 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 90%    188 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     143 GO terms are associated with    188 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 75%    156 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     265 GO terms are associated with    156 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 89%    183 of    205 study items found in association\n",
      " 89%    205 of    230 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     513 GO terms are associated with    183 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 95%    194 of    205 study items found in association\n",
      " 89%    205 of    230 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     186 GO terms are associated with    194 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 79%    161 of    205 study items found in association\n",
      " 89%    205 of    230 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     278 GO terms are associated with    161 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 88%    182 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     474 GO terms are associated with    182 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 94%    196 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     173 GO terms are associated with    196 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 230 IDs ...\n",
      " 73%    152 of    208 study items found in association\n",
      " 90%    208 of    230 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     280 GO terms are associated with    152 of    230 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 90%    198 of    220 study items found in association\n",
      " 92%    220 of    240 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     551 GO terms are associated with    198 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 92%    202 of    220 study items found in association\n",
      " 92%    220 of    240 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     151 GO terms are associated with    202 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 76%    167 of    220 study items found in association\n",
      " 92%    220 of    240 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     329 GO terms are associated with    167 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 86%    188 of    218 study items found in association\n",
      " 91%    218 of    240 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     555 GO terms are associated with    188 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 91%    198 of    218 study items found in association\n",
      " 91%    218 of    240 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     148 GO terms are associated with    198 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 74%    162 of    218 study items found in association\n",
      " 91%    218 of    240 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     280 GO terms are associated with    162 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 89%    192 of    215 study items found in association\n",
      " 90%    215 of    240 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     533 GO terms are associated with    192 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 94%    202 of    215 study items found in association\n",
      " 90%    215 of    240 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     191 GO terms are associated with    202 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 78%    168 of    215 study items found in association\n",
      " 90%    215 of    240 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     294 GO terms are associated with    168 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 87%    188 of    217 study items found in association\n",
      " 90%    217 of    240 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     494 GO terms are associated with    188 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 94%    203 of    217 study items found in association\n",
      " 90%    217 of    240 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     176 GO terms are associated with    203 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 240 IDs ...\n",
      " 74%    160 of    217 study items found in association\n",
      " 90%    217 of    240 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     290 GO terms are associated with    160 of    240 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 90%    208 of    230 study items found in association\n",
      " 92%    230 of    250 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     572 GO terms are associated with    208 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 92%    211 of    230 study items found in association\n",
      " 92%    230 of    250 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     154 GO terms are associated with    211 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 77%    176 of    230 study items found in association\n",
      " 92%    230 of    250 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     337 GO terms are associated with    176 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 85%    194 of    227 study items found in association\n",
      " 91%    227 of    250 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     565 GO terms are associated with    194 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 91%    207 of    227 study items found in association\n",
      " 91%    227 of    250 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     149 GO terms are associated with    207 of    250 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 73%    166 of    227 study items found in association\n",
      " 91%    227 of    250 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     288 GO terms are associated with    166 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 88%    199 of    225 study items found in association\n",
      " 90%    225 of    250 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     546 GO terms are associated with    199 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 94%    212 of    225 study items found in association\n",
      " 90%    225 of    250 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     199 GO terms are associated with    212 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 78%    175 of    225 study items found in association\n",
      " 90%    225 of    250 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     304 GO terms are associated with    175 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 87%    197 of    226 study items found in association\n",
      " 90%    226 of    250 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     511 GO terms are associated with    197 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 94%    212 of    226 study items found in association\n",
      " 90%    226 of    250 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     180 GO terms are associated with    212 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 250 IDs ...\n",
      " 74%    168 of    226 study items found in association\n",
      " 90%    226 of    250 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     301 GO terms are associated with    168 of    250 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 91%    218 of    240 study items found in association\n",
      " 92%    240 of    260 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     588 GO terms are associated with    218 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 92%    221 of    240 study items found in association\n",
      " 92%    240 of    260 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     163 GO terms are associated with    221 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 76%    183 of    240 study items found in association\n",
      " 92%    240 of    260 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     340 GO terms are associated with    183 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 86%    203 of    237 study items found in association\n",
      " 91%    237 of    260 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     581 GO terms are associated with    203 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 91%    216 of    237 study items found in association\n",
      " 91%    237 of    260 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     153 GO terms are associated with    216 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 74%    175 of    237 study items found in association\n",
      " 91%    237 of    260 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     300 GO terms are associated with    175 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 88%    207 of    234 study items found in association\n",
      " 90%    234 of    260 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     563 GO terms are associated with    207 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 94%    221 of    234 study items found in association\n",
      " 90%    234 of    260 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     201 GO terms are associated with    221 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 78%    183 of    234 study items found in association\n",
      " 90%    234 of    260 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     309 GO terms are associated with    183 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 87%    206 of    236 study items found in association\n",
      " 91%    236 of    260 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     532 GO terms are associated with    206 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 94%    221 of    236 study items found in association\n",
      " 91%    236 of    260 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     185 GO terms are associated with    221 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 260 IDs ...\n",
      " 74%    175 of    236 study items found in association\n",
      " 91%    236 of    260 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     315 GO terms are associated with    175 of    260 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 91%    226 of    249 study items found in association\n",
      " 92%    249 of    270 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     603 GO terms are associated with    226 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 92%    230 of    249 study items found in association\n",
      " 92%    249 of    270 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     168 GO terms are associated with    230 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 76%    190 of    249 study items found in association\n",
      " 92%    249 of    270 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     348 GO terms are associated with    190 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 86%    212 of    247 study items found in association\n",
      " 91%    247 of    270 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     595 GO terms are associated with    212 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 91%    226 of    247 study items found in association\n",
      " 91%    247 of    270 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     155 GO terms are associated with    226 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 74%    183 of    247 study items found in association\n",
      " 91%    247 of    270 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     313 GO terms are associated with    183 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 89%    216 of    244 study items found in association\n",
      " 90%    244 of    270 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     572 GO terms are associated with    216 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 95%    231 of    244 study items found in association\n",
      " 90%    244 of    270 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     203 GO terms are associated with    231 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 77%    189 of    244 study items found in association\n",
      " 90%    244 of    270 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     313 GO terms are associated with    189 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 88%    215 of    245 study items found in association\n",
      " 91%    245 of    270 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     551 GO terms are associated with    215 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 94%    230 of    245 study items found in association\n",
      " 91%    245 of    270 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     192 GO terms are associated with    230 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 270 IDs ...\n",
      " 75%    183 of    245 study items found in association\n",
      " 91%    245 of    270 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     318 GO terms are associated with    183 of    270 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 90%    233 of    258 study items found in association\n",
      " 92%    258 of    280 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     618 GO terms are associated with    233 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 92%    238 of    258 study items found in association\n",
      " 92%    258 of    280 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     173 GO terms are associated with    238 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 76%    197 of    258 study items found in association\n",
      " 92%    258 of    280 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     362 GO terms are associated with    197 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 86%    219 of    256 study items found in association\n",
      " 91%    256 of    280 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     616 GO terms are associated with    219 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 92%    235 of    256 study items found in association\n",
      " 91%    256 of    280 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     162 GO terms are associated with    235 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 74%    189 of    256 study items found in association\n",
      " 91%    256 of    280 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     325 GO terms are associated with    189 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 88%    224 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     595 GO terms are associated with    224 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 94%    239 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     206 GO terms are associated with    239 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 78%    197 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     327 GO terms are associated with    197 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 87%    222 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     558 GO terms are associated with    222 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 94%    238 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     198 GO terms are associated with    238 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 280 IDs ...\n",
      " 74%    188 of    254 study items found in association\n",
      " 91%    254 of    280 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     328 GO terms are associated with    188 of    280 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 91%    243 of    268 study items found in association\n",
      " 92%    268 of    290 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     638 GO terms are associated with    243 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 93%    248 of    268 study items found in association\n",
      " 92%    268 of    290 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     176 GO terms are associated with    248 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 76%    204 of    268 study items found in association\n",
      " 92%    268 of    290 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     368 GO terms are associated with    204 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 86%    227 of    265 study items found in association\n",
      " 91%    265 of    290 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     634 GO terms are associated with    227 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 92%    244 of    265 study items found in association\n",
      " 91%    265 of    290 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     165 GO terms are associated with    244 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 74%    195 of    265 study items found in association\n",
      " 91%    265 of    290 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     334 GO terms are associated with    195 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 88%    232 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     613 GO terms are associated with    232 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 94%    248 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     207 GO terms are associated with    248 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 78%    205 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     335 GO terms are associated with    205 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 87%    230 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     566 GO terms are associated with    230 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 94%    248 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     201 GO terms are associated with    248 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 290 IDs ...\n",
      " 74%    195 of    264 study items found in association\n",
      " 91%    264 of    290 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     333 GO terms are associated with    195 of    290 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 91%    251 of    276 study items found in association\n",
      " 92%    276 of    300 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     657 GO terms are associated with    251 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 93%    256 of    276 study items found in association\n",
      " 92%    276 of    300 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     179 GO terms are associated with    256 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 76%    210 of    276 study items found in association\n",
      " 92%    276 of    300 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     379 GO terms are associated with    210 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 85%    233 of    274 study items found in association\n",
      " 91%    274 of    300 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     654 GO terms are associated with    233 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 92%    253 of    274 study items found in association\n",
      " 91%    274 of    300 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     166 GO terms are associated with    253 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 74%    203 of    274 study items found in association\n",
      " 91%    274 of    300 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     345 GO terms are associated with    203 of    300 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 88%    240 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     625 GO terms are associated with    240 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 94%    257 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     212 GO terms are associated with    257 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 77%    211 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     340 GO terms are associated with    211 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 88%    239 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     579 GO terms are associated with    239 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 94%    257 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     208 GO terms are associated with    257 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 300 IDs ...\n",
      " 74%    203 of    273 study items found in association\n",
      " 91%    273 of    300 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     341 GO terms are associated with    203 of    300 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 91%    258 of    285 study items found in association\n",
      " 92%    285 of    310 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     667 GO terms are associated with    258 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 93%    264 of    285 study items found in association\n",
      " 92%    285 of    310 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     182 GO terms are associated with    264 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 76%    217 of    285 study items found in association\n",
      " 92%    285 of    310 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     389 GO terms are associated with    217 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 86%    243 of    284 study items found in association\n",
      " 92%    284 of    310 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     679 GO terms are associated with    243 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 93%    263 of    284 study items found in association\n",
      " 92%    284 of    310 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     168 GO terms are associated with    263 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 75%    213 of    284 study items found in association\n",
      " 92%    284 of    310 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     359 GO terms are associated with    213 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 88%    246 of    281 study items found in association\n",
      " 91%    281 of    310 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     629 GO terms are associated with    246 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 94%    265 of    281 study items found in association\n",
      " 91%    281 of    310 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     213 GO terms are associated with    265 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 77%    217 of    281 study items found in association\n",
      " 91%    281 of    310 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     346 GO terms are associated with    217 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 87%    246 of    282 study items found in association\n",
      " 91%    282 of    310 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     587 GO terms are associated with    246 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 94%    266 of    282 study items found in association\n",
      " 91%    282 of    310 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     210 GO terms are associated with    266 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 310 IDs ...\n",
      " 74%    210 of    282 study items found in association\n",
      " 91%    282 of    310 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     350 GO terms are associated with    210 of    310 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 90%    264 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     680 GO terms are associated with    264 of    319 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 92%    271 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     186 GO terms are associated with    271 of    319 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 76%    222 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     396 GO terms are associated with    222 of    319 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 86%    251 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     687 GO terms are associated with    251 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 92%    271 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     169 GO terms are associated with    271 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 74%    217 of    293 study items found in association\n",
      " 92%    293 of    320 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     365 GO terms are associated with    217 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 87%    253 of    290 study items found in association\n",
      " 91%    290 of    320 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     655 GO terms are associated with    253 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 94%    273 of    290 study items found in association\n",
      " 91%    290 of    320 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     217 GO terms are associated with    273 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 77%    223 of    290 study items found in association\n",
      " 91%    290 of    320 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     355 GO terms are associated with    223 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 88%    256 of    292 study items found in association\n",
      " 91%    292 of    320 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     605 GO terms are associated with    256 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 94%    275 of    292 study items found in association\n",
      " 91%    292 of    320 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     214 GO terms are associated with    275 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 320 IDs ...\n",
      " 75%    220 of    292 study items found in association\n",
      " 91%    292 of    320 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     359 GO terms are associated with    220 of    320 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 90%    272 of    303 study items found in association\n",
      " 92%    303 of    330 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     685 GO terms are associated with    272 of    329 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 92%    280 of    303 study items found in association\n",
      " 92%    303 of    330 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     187 GO terms are associated with    280 of    329 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 76%    229 of    303 study items found in association\n",
      " 92%    303 of    330 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     399 GO terms are associated with    229 of    329 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 85%    255 of    301 study items found in association\n",
      " 91%    301 of    330 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     691 GO terms are associated with    255 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 92%    276 of    301 study items found in association\n",
      " 91%    301 of    330 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     170 GO terms are associated with    276 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 73%    221 of    301 study items found in association\n",
      " 91%    301 of    330 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     366 GO terms are associated with    221 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 87%    260 of    298 study items found in association\n",
      " 90%    298 of    330 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     669 GO terms are associated with    260 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 94%    281 of    298 study items found in association\n",
      " 90%    298 of    330 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     219 GO terms are associated with    281 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 77%    230 of    298 study items found in association\n",
      " 90%    298 of    330 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     360 GO terms are associated with    230 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 87%    263 of    302 study items found in association\n",
      " 92%    302 of    330 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     620 GO terms are associated with    263 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 94%    283 of    302 study items found in association\n",
      " 92%    302 of    330 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     217 GO terms are associated with    283 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 330 IDs ...\n",
      " 75%    227 of    302 study items found in association\n",
      " 92%    302 of    330 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     367 GO terms are associated with    227 of    330 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 90%    280 of    311 study items found in association\n",
      " 91%    311 of    340 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     704 GO terms are associated with    280 of    339 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 93%    288 of    311 study items found in association\n",
      " 91%    311 of    340 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     195 GO terms are associated with    288 of    339 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 76%    237 of    311 study items found in association\n",
      " 91%    311 of    340 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     405 GO terms are associated with    237 of    339 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 85%    262 of    310 study items found in association\n",
      " 91%    310 of    340 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     710 GO terms are associated with    262 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 92%    285 of    310 study items found in association\n",
      " 91%    310 of    340 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     177 GO terms are associated with    285 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 74%    228 of    310 study items found in association\n",
      " 91%    310 of    340 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     371 GO terms are associated with    228 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 87%    269 of    308 study items found in association\n",
      " 91%    308 of    340 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     676 GO terms are associated with    269 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 94%    291 of    308 study items found in association\n",
      " 91%    308 of    340 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     229 GO terms are associated with    291 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 77%    236 of    308 study items found in association\n",
      " 91%    308 of    340 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     366 GO terms are associated with    236 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 88%    273 of    312 study items found in association\n",
      " 92%    312 of    340 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     634 GO terms are associated with    273 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 94%    292 of    312 study items found in association\n",
      " 92%    312 of    340 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     219 GO terms are associated with    292 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 340 IDs ...\n",
      " 76%    236 of    312 study items found in association\n",
      " 92%    312 of    340 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     373 GO terms are associated with    236 of    340 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 89%    285 of    320 study items found in association\n",
      " 91%    320 of    350 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     713 GO terms are associated with    285 of    349 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 92%    293 of    320 study items found in association\n",
      " 91%    320 of    350 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     199 GO terms are associated with    293 of    349 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 76%    242 of    320 study items found in association\n",
      " 91%    320 of    350 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     412 GO terms are associated with    242 of    349 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 84%    267 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     711 GO terms are associated with    267 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 92%    292 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     180 GO terms are associated with    292 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 73%    232 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     372 GO terms are associated with    232 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 87%    278 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     689 GO terms are associated with    278 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 95%    301 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     236 GO terms are associated with    301 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 77%    244 of    318 study items found in association\n",
      " 91%    318 of    350 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     378 GO terms are associated with    244 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 88%    280 of    319 study items found in association\n",
      " 91%    319 of    350 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     649 GO terms are associated with    280 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 94%    299 of    319 study items found in association\n",
      " 91%    319 of    350 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     220 GO terms are associated with    299 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 350 IDs ...\n",
      " 76%    242 of    319 study items found in association\n",
      " 91%    319 of    350 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     378 GO terms are associated with    242 of    350 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 89%    295 of    330 study items found in association\n",
      " 92%    330 of    360 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     727 GO terms are associated with    295 of    359 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 92%    303 of    330 study items found in association\n",
      " 92%    330 of    360 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     203 GO terms are associated with    303 of    359 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 76%    251 of    330 study items found in association\n",
      " 92%    330 of    360 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     430 GO terms are associated with    251 of    359 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 84%    274 of    326 study items found in association\n",
      " 91%    326 of    360 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     721 GO terms are associated with    274 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 92%    300 of    326 study items found in association\n",
      " 91%    326 of    360 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     180 GO terms are associated with    300 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 73%    237 of    326 study items found in association\n",
      " 91%    326 of    360 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     377 GO terms are associated with    237 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 88%    287 of    328 study items found in association\n",
      " 91%    328 of    360 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     712 GO terms are associated with    287 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 94%    309 of    328 study items found in association\n",
      " 91%    328 of    360 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     240 GO terms are associated with    309 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 77%    253 of    328 study items found in association\n",
      " 91%    328 of    360 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     389 GO terms are associated with    253 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 88%    289 of    329 study items found in association\n",
      " 91%    329 of    360 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     656 GO terms are associated with    289 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 94%    309 of    329 study items found in association\n",
      " 91%    329 of    360 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     227 GO terms are associated with    309 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 360 IDs ...\n",
      " 76%    251 of    329 study items found in association\n",
      " 91%    329 of    360 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     385 GO terms are associated with    251 of    360 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 89%    303 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     740 GO terms are associated with    303 of    369 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 91%    309 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     207 GO terms are associated with    309 of    369 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 76%    258 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     438 GO terms are associated with    258 of    369 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 84%    281 of    334 study items found in association\n",
      " 90%    334 of    370 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     726 GO terms are associated with    281 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 92%    307 of    334 study items found in association\n",
      " 90%    334 of    370 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     184 GO terms are associated with    307 of    370 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 72%    242 of    334 study items found in association\n",
      " 90%    334 of    370 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     378 GO terms are associated with    242 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 88%    297 of    338 study items found in association\n",
      " 91%    338 of    370 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     728 GO terms are associated with    297 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 94%    319 of    338 study items found in association\n",
      " 91%    338 of    370 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     240 GO terms are associated with    319 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 78%    262 of    338 study items found in association\n",
      " 91%    338 of    370 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     397 GO terms are associated with    262 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 88%    298 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     664 GO terms are associated with    298 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 94%    318 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     234 GO terms are associated with    318 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 370 IDs ...\n",
      " 76%    259 of    339 study items found in association\n",
      " 92%    339 of    370 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     390 GO terms are associated with    259 of    370 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 89%    311 of    349 study items found in association\n",
      " 92%    349 of    380 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     752 GO terms are associated with    311 of    379 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 91%    316 of    349 study items found in association\n",
      " 92%    349 of    380 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     208 GO terms are associated with    316 of    379 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 76%    264 of    349 study items found in association\n",
      " 92%    349 of    380 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     444 GO terms are associated with    264 of    379 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 84%    290 of    344 study items found in association\n",
      " 91%    344 of    380 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     728 GO terms are associated with    290 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 92%    317 of    344 study items found in association\n",
      " 91%    344 of    380 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     186 GO terms are associated with    317 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 72%    248 of    344 study items found in association\n",
      " 91%    344 of    380 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     382 GO terms are associated with    248 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 88%    306 of    347 study items found in association\n",
      " 91%    347 of    380 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     739 GO terms are associated with    306 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 95%    328 of    347 study items found in association\n",
      " 91%    347 of    380 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     243 GO terms are associated with    328 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 78%    270 of    347 study items found in association\n",
      " 91%    347 of    380 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     406 GO terms are associated with    270 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 88%    307 of    348 study items found in association\n",
      " 92%    348 of    380 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     694 GO terms are associated with    307 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 94%    327 of    348 study items found in association\n",
      " 92%    348 of    380 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     240 GO terms are associated with    327 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 380 IDs ...\n",
      " 77%    268 of    348 study items found in association\n",
      " 92%    348 of    380 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     398 GO terms are associated with    268 of    380 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 88%    316 of    358 study items found in association\n",
      " 92%    358 of    390 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     761 GO terms are associated with    316 of    389 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 91%    324 of    358 study items found in association\n",
      " 92%    358 of    390 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     212 GO terms are associated with    324 of    389 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 75%    268 of    358 study items found in association\n",
      " 92%    358 of    390 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     448 GO terms are associated with    268 of    389 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 85%    298 of    352 study items found in association\n",
      " 90%    352 of    390 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     740 GO terms are associated with    298 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 92%    325 of    352 study items found in association\n",
      " 90%    352 of    390 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     195 GO terms are associated with    325 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 72%    252 of    352 study items found in association\n",
      " 90%    352 of    390 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     388 GO terms are associated with    252 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 88%    314 of    357 study items found in association\n",
      " 92%    357 of    390 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     757 GO terms are associated with    314 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 94%    337 of    357 study items found in association\n",
      " 92%    357 of    390 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     247 GO terms are associated with    337 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 78%    277 of    357 study items found in association\n",
      " 92%    357 of    390 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     412 GO terms are associated with    277 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 88%    315 of    356 study items found in association\n",
      " 91%    356 of    390 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     704 GO terms are associated with    315 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 94%    335 of    356 study items found in association\n",
      " 91%    356 of    390 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     242 GO terms are associated with    335 of    390 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 390 IDs ...\n",
      " 77%    275 of    356 study items found in association\n",
      " 91%    356 of    390 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     402 GO terms are associated with    275 of    390 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 89%    324 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     775 GO terms are associated with    324 of    399 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 91%    332 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     217 GO terms are associated with    332 of    399 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 75%    274 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     448 GO terms are associated with    274 of    399 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 85%    306 of    362 study items found in association\n",
      " 90%    362 of    400 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     752 GO terms are associated with    306 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 92%    334 of    362 study items found in association\n",
      " 90%    362 of    400 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     198 GO terms are associated with    334 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 72%    259 of    362 study items found in association\n",
      " 90%    362 of    400 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     394 GO terms are associated with    259 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 88%    323 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     773 GO terms are associated with    323 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 95%    346 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     252 GO terms are associated with    346 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 78%    285 of    366 study items found in association\n",
      " 92%    366 of    400 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     413 GO terms are associated with    285 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 89%    324 of    365 study items found in association\n",
      " 91%    365 of    400 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     718 GO terms are associated with    324 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 94%    344 of    365 study items found in association\n",
      " 91%    365 of    400 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     247 GO terms are associated with    344 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 400 IDs ...\n",
      " 78%    283 of    365 study items found in association\n",
      " 91%    365 of    400 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     408 GO terms are associated with    283 of    400 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 89%    332 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     789 GO terms are associated with    332 of    409 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 91%    341 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     219 GO terms are associated with    341 of    409 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 74%    279 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     453 GO terms are associated with    279 of    409 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 85%    315 of    372 study items found in association\n",
      " 91%    372 of    410 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     768 GO terms are associated with    315 of    410 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 92%    343 of    372 study items found in association\n",
      " 91%    372 of    410 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     201 GO terms are associated with    343 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 72%    267 of    372 study items found in association\n",
      " 91%    372 of    410 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     398 GO terms are associated with    267 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 89%    332 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     780 GO terms are associated with    332 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 95%    355 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     259 GO terms are associated with    355 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 78%    294 of    375 study items found in association\n",
      " 91%    375 of    410 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     422 GO terms are associated with    294 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 89%    332 of    374 study items found in association\n",
      " 91%    374 of    410 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     728 GO terms are associated with    332 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 94%    351 of    374 study items found in association\n",
      " 91%    374 of    410 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     249 GO terms are associated with    351 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 410 IDs ...\n",
      " 78%    290 of    374 study items found in association\n",
      " 91%    374 of    410 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     418 GO terms are associated with    290 of    410 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 89%    341 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     808 GO terms are associated with    341 of    419 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 91%    350 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     221 GO terms are associated with    350 of    419 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 74%    286 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     464 GO terms are associated with    286 of    419 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 85%    324 of    382 study items found in association\n",
      " 91%    382 of    420 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     776 GO terms are associated with    324 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 92%    353 of    382 study items found in association\n",
      " 91%    382 of    420 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     203 GO terms are associated with    353 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 72%    275 of    382 study items found in association\n",
      " 91%    382 of    420 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     409 GO terms are associated with    275 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 89%    341 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     787 GO terms are associated with    341 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 95%    364 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     263 GO terms are associated with    364 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 78%    299 of    385 study items found in association\n",
      " 92%    385 of    420 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     424 GO terms are associated with    299 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 88%    338 of    384 study items found in association\n",
      " 91%    384 of    420 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     740 GO terms are associated with    338 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 93%    359 of    384 study items found in association\n",
      " 91%    384 of    420 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     252 GO terms are associated with    359 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 420 IDs ...\n",
      " 77%    296 of    384 study items found in association\n",
      " 91%    384 of    420 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     427 GO terms are associated with    296 of    420 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 89%    350 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     824 GO terms are associated with    350 of    429 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 91%    360 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     224 GO terms are associated with    360 of    429 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 74%    293 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     475 GO terms are associated with    293 of    429 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 85%    330 of    390 study items found in association\n",
      " 91%    390 of    430 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     787 GO terms are associated with    330 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 92%    360 of    390 study items found in association\n",
      " 91%    390 of    430 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     203 GO terms are associated with    360 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 72%    281 of    390 study items found in association\n",
      " 91%    390 of    430 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     417 GO terms are associated with    281 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 89%    350 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     799 GO terms are associated with    350 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 95%    374 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     264 GO terms are associated with    374 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 78%    307 of    395 study items found in association\n",
      " 92%    395 of    430 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     434 GO terms are associated with    307 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 88%    347 of    394 study items found in association\n",
      " 92%    394 of    430 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     766 GO terms are associated with    347 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 93%    367 of    394 study items found in association\n",
      " 92%    394 of    430 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     259 GO terms are associated with    367 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 430 IDs ...\n",
      " 77%    304 of    394 study items found in association\n",
      " 92%    394 of    430 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     437 GO terms are associated with    304 of    430 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 89%    359 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     832 GO terms are associated with    359 of    439 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 91%    369 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     224 GO terms are associated with    369 of    439 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 74%    300 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     483 GO terms are associated with    300 of    439 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 85%    339 of    400 study items found in association\n",
      " 91%    400 of    440 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     794 GO terms are associated with    339 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 92%    370 of    400 study items found in association\n",
      " 91%    400 of    440 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     204 GO terms are associated with    370 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 72%    288 of    400 study items found in association\n",
      " 91%    400 of    440 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     422 GO terms are associated with    288 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 89%    358 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     811 GO terms are associated with    358 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 95%    383 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     265 GO terms are associated with    383 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 78%    315 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     441 GO terms are associated with    315 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 88%    356 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     776 GO terms are associated with    356 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 93%    375 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     264 GO terms are associated with    375 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 440 IDs ...\n",
      " 76%    309 of    404 study items found in association\n",
      " 92%    404 of    440 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     440 GO terms are associated with    309 of    440 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 89%    369 of    414 study items found in association\n",
      " 92%    414 of    450 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     849 GO terms are associated with    369 of    449 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 92%    379 of    414 study items found in association\n",
      " 92%    414 of    450 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     226 GO terms are associated with    379 of    449 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 74%    307 of    414 study items found in association\n",
      " 92%    414 of    450 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     492 GO terms are associated with    307 of    449 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 85%    348 of    410 study items found in association\n",
      " 91%    410 of    450 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     805 GO terms are associated with    348 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 92%    379 of    410 study items found in association\n",
      " 91%    410 of    450 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     210 GO terms are associated with    379 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 72%    296 of    410 study items found in association\n",
      " 91%    410 of    450 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     434 GO terms are associated with    296 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 88%    364 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     817 GO terms are associated with    364 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 95%    391 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     267 GO terms are associated with    391 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 77%    320 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     442 GO terms are associated with    320 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 88%    364 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     796 GO terms are associated with    364 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 93%    384 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     268 GO terms are associated with    384 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 450 IDs ...\n",
      " 76%    315 of    413 study items found in association\n",
      " 92%    413 of    450 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     452 GO terms are associated with    315 of    450 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 89%    374 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     853 GO terms are associated with    374 of    459 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 91%    384 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     227 GO terms are associated with    384 of    459 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 74%    314 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     498 GO terms are associated with    314 of    459 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 85%    358 of    420 study items found in association\n",
      " 91%    420 of    460 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     815 GO terms are associated with    358 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 92%    387 of    420 study items found in association\n",
      " 91%    420 of    460 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     212 GO terms are associated with    387 of    460 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 73%    306 of    420 study items found in association\n",
      " 91%    420 of    460 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     441 GO terms are associated with    306 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 88%    372 of    423 study items found in association\n",
      " 92%    423 of    460 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     829 GO terms are associated with    372 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 94%    399 of    423 study items found in association\n",
      " 92%    423 of    460 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     269 GO terms are associated with    399 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 77%    327 of    423 study items found in association\n",
      " 92%    423 of    460 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     453 GO terms are associated with    327 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 88%    372 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     810 GO terms are associated with    372 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 93%    393 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     271 GO terms are associated with    393 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 460 IDs ...\n",
      " 77%    323 of    422 study items found in association\n",
      " 92%    422 of    460 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     459 GO terms are associated with    323 of    460 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 88%    382 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     868 GO terms are associated with    382 of    469 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 90%    390 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     229 GO terms are associated with    390 of    469 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 74%    320 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     505 GO terms are associated with    320 of    469 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 85%    366 of    430 study items found in association\n",
      " 91%    430 of    470 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     836 GO terms are associated with    366 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 92%    396 of    430 study items found in association\n",
      " 91%    430 of    470 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     214 GO terms are associated with    396 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 73%    315 of    430 study items found in association\n",
      " 91%    430 of    470 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     449 GO terms are associated with    315 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 88%    379 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     838 GO terms are associated with    379 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 94%    407 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     271 GO terms are associated with    407 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 77%    334 of    432 study items found in association\n",
      " 92%    432 of    470 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     461 GO terms are associated with    334 of    470 study items\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 88%    381 of    431 study items found in association\n",
      " 92%    431 of    470 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     825 GO terms are associated with    381 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 93%    402 of    431 study items found in association\n",
      " 92%    431 of    470 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     275 GO terms are associated with    402 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 470 IDs ...\n",
      " 77%    330 of    431 study items found in association\n",
      " 92%    431 of    470 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     467 GO terms are associated with    330 of    470 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 89%    391 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     884 GO terms are associated with    391 of    479 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 90%    399 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     236 GO terms are associated with    399 of    479 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 74%    326 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     508 GO terms are associated with    326 of    479 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 85%    372 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     849 GO terms are associated with    372 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 92%    406 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     221 GO terms are associated with    406 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 73%    321 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     453 GO terms are associated with    321 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 88%    388 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     853 GO terms are associated with    388 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 94%    416 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     274 GO terms are associated with    416 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 78%    342 of    441 study items found in association\n",
      " 92%    441 of    480 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     467 GO terms are associated with    342 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 88%    389 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     839 GO terms are associated with    389 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 93%    410 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     277 GO terms are associated with    410 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 480 IDs ...\n",
      " 76%    336 of    440 study items found in association\n",
      " 92%    440 of    480 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     477 GO terms are associated with    336 of    480 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 88%    396 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     892 GO terms are associated with    396 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 91%    409 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     238 GO terms are associated with    409 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 73%    331 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     511 GO terms are associated with    331 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 84%    379 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     859 GO terms are associated with    379 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 92%    414 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     222 GO terms are associated with    414 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 73%    326 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     457 GO terms are associated with    326 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 88%    398 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     862 GO terms are associated with    398 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 94%    426 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     279 GO terms are associated with    426 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 77%    349 of    451 study items found in association\n",
      " 92%    451 of    490 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     474 GO terms are associated with    349 of    490 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 88%    396 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     853 GO terms are associated with    396 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 93%    417 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     281 GO terms are associated with    417 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 490 IDs ...\n",
      " 76%    341 of    449 study items found in association\n",
      " 92%    449 of    490 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     482 GO terms are associated with    341 of    489 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    405 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     902 GO terms are associated with    405 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 91%    418 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     238 GO terms are associated with    418 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 73%    338 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     516 GO terms are associated with    338 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 84%    387 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     876 GO terms are associated with    387 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 92%    423 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     224 GO terms are associated with    423 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 73%    333 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     466 GO terms are associated with    333 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    406 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     872 GO terms are associated with    406 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 94%    435 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     281 GO terms are associated with    435 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 77%    356 of    461 study items found in association\n",
      " 92%    461 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     479 GO terms are associated with    356 of    500 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 88%    404 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     861 GO terms are associated with    404 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 93%    426 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     282 GO terms are associated with    426 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 500 IDs ...\n",
      " 76%    349 of    458 study items found in association\n",
      " 92%    458 of    500 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     489 GO terms are associated with    349 of    499 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n"
     ]
    }
   ],
   "source": [
    "# Function that returns the number of significant p-values according to the fdr_bh test:\n",
    "def goea_return_nsig(data, group_name):\n",
    "    goea_results_all = goeaobj.run_study(list(data[group_name]))\n",
    "    goea_results_sig = [r for r in goea_results_all if r.p_fdr_bh < 0.05]\n",
    "    return len(goea_results_sig)\n",
    "\n",
    "# Call the function in a loop that goes through all group sizes (in multiples of 10) and records the number of significant GO terms:\n",
    "nsig = list()\n",
    "for n in range(10, 501, 10):\n",
    "    nt = goea_return_nsig(data.head(n), \"top_redness\")\n",
    "    ntn = goea_return_nsig(data.head(n), \"top_norm_redness\")\n",
    "    nb = goea_return_nsig(data.head(n), \"bottom_redness\")\n",
    "    nbn = goea_return_nsig(data.head(n), \"bottom_norm_redness\")\n",
    "    nsig.append([n, nt, ntn, nb, nbn])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[10, 0, 0, 0, 0],\n",
       " [20, 0, 0, 0, 0],\n",
       " [30, 0, 0, 0, 0],\n",
       " [40, 0, 0, 0, 0],\n",
       " [50, 0, 0, 0, 0],\n",
       " [60, 0, 0, 0, 0],\n",
       " [70, 0, 0, 0, 0],\n",
       " [80, 0, 0, 0, 0],\n",
       " [90, 0, 0, 0, 0],\n",
       " [100, 0, 0, 0, 0],\n",
       " [110, 0, 0, 0, 0],\n",
       " [120, 0, 0, 0, 0],\n",
       " [130, 0, 0, 0, 0],\n",
       " [140, 0, 0, 0, 0],\n",
       " [150, 2, 0, 0, 0],\n",
       " [160, 1, 0, 0, 0],\n",
       " [170, 1, 0, 0, 0],\n",
       " [180, 1, 0, 0, 0],\n",
       " [190, 1, 0, 0, 0],\n",
       " [200, 1, 0, 0, 0],\n",
       " [210, 0, 0, 0, 0],\n",
       " [220, 0, 0, 0, 0],\n",
       " [230, 0, 0, 0, 0],\n",
       " [240, 0, 0, 0, 0],\n",
       " [250, 0, 0, 0, 0],\n",
       " [260, 0, 0, 0, 0],\n",
       " [270, 0, 0, 0, 0],\n",
       " [280, 0, 0, 0, 0],\n",
       " [290, 0, 0, 0, 0],\n",
       " [300, 0, 0, 0, 0],\n",
       " [310, 0, 0, 0, 0],\n",
       " [320, 0, 0, 0, 0],\n",
       " [330, 0, 0, 0, 0],\n",
       " [340, 0, 0, 0, 0],\n",
       " [350, 0, 0, 0, 0],\n",
       " [360, 0, 0, 0, 0],\n",
       " [370, 0, 0, 0, 0],\n",
       " [380, 0, 0, 0, 0],\n",
       " [390, 0, 0, 0, 0],\n",
       " [400, 0, 0, 0, 0],\n",
       " [410, 0, 0, 0, 0],\n",
       " [420, 0, 0, 0, 0],\n",
       " [430, 0, 0, 0, 0],\n",
       " [440, 0, 0, 0, 0],\n",
       " [450, 0, 0, 0, 0],\n",
       " [460, 0, 0, 0, 0],\n",
       " [470, 0, 0, 0, 0],\n",
       " [480, 0, 0, 0, 0],\n",
       " [490, 0, 0, 0, 0],\n",
       " [500, 0, 0, 0, 0]]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nsig"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So there's actually a few group sizes in which the top redness is enriched. Let's save those results:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 91%    130 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     414 GO terms are associated with    130 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       4 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 93%    133 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     113 GO terms are associated with    133 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 79%    113 of    143 study items found in association\n",
      " 95%    143 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     246 GO terms are associated with    113 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       1 GO terms found significant (< 0.05=alpha) (  1 enriched +   0 purified): statsmodels fdr_bh\n",
      "       5 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "    118 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_redness.tsv\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 90%    122 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     405 GO terms are associated with    122 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 93%    125 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     122 GO terms are associated with    125 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 79%    106 of    135 study items found in association\n",
      " 90%    135 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     202 GO terms are associated with    106 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "    109 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_top_norm_redness.tsv\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 88%    120 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     366 GO terms are associated with    120 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 96%    130 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     152 GO terms are associated with    130 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 77%    105 of    136 study items found in association\n",
      " 91%    136 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     190 GO terms are associated with    105 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     89 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_redness.tsv\n",
      "\n",
      "Run BP Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 85%    117 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 2,845 uncorrected p-values using fisher\n",
      "   2,845 GO terms are associated with  3,845 of  4,436 population items\n",
      "     332 GO terms are associated with    117 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run CC Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 94%    129 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 712 uncorrected p-values using fisher\n",
      "     712 GO terms are associated with  4,099 of  4,436 population items\n",
      "     139 GO terms are associated with    129 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "\n",
      "Run MF Gene Ontology Analysis: current study set of 150 IDs ...\n",
      " 70%     96 of    137 study items found in association\n",
      " 91%    137 of    150 study items found in population(4436)\n",
      "Calculating 1,793 uncorrected p-values using fisher\n",
      "   1,793 GO terms are associated with  3,299 of  4,436 population items\n",
      "     193 GO terms are associated with     96 of    150 study items\n",
      "  METHOD fdr_bh:\n",
      "       0 GO terms found significant (< 0.05=alpha) (  0 enriched +   0 purified): statsmodels fdr_bh\n",
      "       0 study items associated with significant GO IDs (enriched)\n",
      "       0 study items associated with significant GO IDs (purified)\n",
      "     80 items WROTE: C:\\Users\\bejsab\\Documents\\repos\\phenobooth-analysis\\data\\final\\p_values_bottom_norm_redness.tsv\n"
     ]
    }
   ],
   "source": [
    "goea(data.head(150), \"top_redness\")\n",
    "goea(data.head(150), \"top_norm_redness\")\n",
    "goea(data.head(150), \"bottom_redness\")\n",
    "goea(data.head(150), \"bottom_norm_redness\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
